1Password/1Password.jss.recipe

```diff
@@ -17,7 +17,7 @@
 		<key>NAME</key>
 		<string>1Password</string>
 		<key>OS_REQUIREMENTS</key>
-		<string>10.14.x,10.13.x,10.12.6</string>
+		<string>10.15.x,10.14.x,10.13.x,10.12.6</string>
```

Comment:
According to the 1Password website, the latest version requires 10.13+.
<img width="427" alt="image" src="https://user-images.githubusercontent.com/7801391/71314415-c316bf00-23fc-11ea-8323-bfd82b0bdb23.png">